src/bin/update_snapshots/main.rs

```diff
@@ -20,11 +20,18 @@ use crate::{
 fn main() {
     println!("Updating snapshots:");
 
-    let test: Vec<_> = snapshot_tests()
-        .into_iter()
-        .map(TestCaseInstance::new)
-        .collect();
-    for test in test.iter() {
+    let tests: Vec<_> = snapshot_tests();
+    let has_only_flag = tests.iter().any(|t| t.only);
```

Comment:
This flag is intended for local debuging with test. If you want to run only one test you can set `only: true` on there and run `cargo --bin update_snapshots`. Only selected test will be run and code responsible for removing unused snapshots will not be run.